Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -286,6 +286,7 @@ def get_threats_request(self, content_hash=None, mitigation_status=None, created
             siteIds=site_ids,
             rank=int(rank) if rank else None,
             keys_to_ignore=keys_to_ignore,
+            incidentStatuses=argToList(incidentStatuses)
```

Comment:
```suggestion
            incident_statuses=argToList(incidentStatuses)
```
shouldn't this be camel cased? similar to other naming conventions?
please keep the convention same as before